docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -2390,8 +2390,8 @@ impenetrable peer groups forming. The default is `false`.
     remote-connections-limit-enabled=false
     ```
 
-Enables or disables limiting the percentage of remote P2P connections initiated by peers. The
-default is true.
+Enables or disables [limiting the percentage of remote P2P connections initiated by peers](#remote-connections-max-percentage).
```

Comment:
Maybe indicate explicitly the option name that is enabled by this flag. It first look like this option is dealing with a percentage, but then, it defaults to "true" which means it's a boolean... so you read the sentence again and oh yeah, it's just enabling the one that uses a percentage... maybe we can clarify.